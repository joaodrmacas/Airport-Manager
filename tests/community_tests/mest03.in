a pDL Portugal Ponta Delgada
a PDl Portugal Ponta Delgada
a LND Inglaterra Londres
a gtm UnitedStates Gotham City
q
