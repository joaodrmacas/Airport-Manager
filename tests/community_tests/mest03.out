invalid airport ID
invalid airport ID
airport LND
invalid airport ID
